content/docs/next-release/frontend-ui.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Frontend/UI
+widescreen: true
+hasparent: true
+weight: 10
+---
+
+## Query Service & UI
+
+**jaeger-query** serves the API endpoints and a React/Javascript UI.
+The service is stateless and is typically run behind a load balancer, e.g. nginx.
+
+At default settings the query service exposes the following port(s):
+
+Port  | Protocol | Function
+----- | -------  | ---
+16686 | HTTP     | **/api/*** endpoints and Jaeger UI at **/**
+16687 | HTTP     | Health check at **/**
```

Comment:
please don't remove this section above from the deployment doc. It refers to query-service, a microservice in Jaeger, and logically belongs to deployment. The rest of this doc refers to customization / configuration of the UI.

Reply:
I added a commit with the change that I wanted.